glotaran/test/test_spectral_decay.py

```diff
@@ -7,6 +7,50 @@
 from glotaran.io import load_parameters
 from glotaran.project import Scheme
 
+MODEL_3C_NO_IRF = """\
+dataset:
+    dataset1: &dataset1
```

Comment:
```suggestion
    dataset1:
```
The anchor isn't used, so there is no need to define it.